docs/src/pages/components/cdk-components/footer/api.jsx

```diff
@@ -68,7 +68,7 @@ const alertPropsTable = () => {
         <td>margin: string</td>
         <td></td>
         <td>
-          Size of the top margin to be applied to the footer ('xxsmall' |
```

Comment:
We should maintain `top margin` comment.

Reply:
Done